newIDE/app/src/MainFrame/EditorContainers/HomePage/LearnSection/EducationCurriculumLesson.js

```diff
@@ -181,7 +181,7 @@ const EducationCurriculumLesson = ({
           <div style={{ ...styles.textContainer, opacity: isLocked ? 0.6 : 1 }}>
             <ColumnStackLayout justifyContent="space-between" noMargin expand>
               <ColumnStackLayout noMargin expand>
-                {!isMobile && !isLandscape && title}
+                {!(isMobile && !isLandscape) && title}
                 {gameLink &&
                   isMobile &&
                   !isLandscape &&
```

Comment:
you seem to be using this `isMobile && !isLandscape` twice, you could make it a variable